nifi-maven-archetypes/nifi-processor-bundle-archetype/src/main/resources/archetype-resources/nifi-__artifactBaseName__-processors/src/main/java/MyProcessor.java

```diff
@@ -41,7 +51,8 @@
 public class MyProcessor extends AbstractProcessor {
 
     public static final PropertyDescriptor MY_PROPERTY = new PropertyDescriptor
-            .Builder().name("My Property")
+            .Builder().name("MY_PROPERTY")
```

Comment:
Hmmmm... luckily you are democratic enough to review my code with those ugly capital letters used on "friendly" names. :-)